packages/run-protocol/src/vaultFactory/vault.js

```diff
@@ -25,27 +25,28 @@ const trace = makeTracer('IV');
 /**
  * @file This has most of the logic for a Vault, to borrow RUN against collateral.
  *
- * The logic here is for InnerVault which is the majority of logic of vaults but
- * the user view is the `vault` value contained in VaultKit.
+ *   The logic here is for InnerVault which is the majority of logic of vaults but
+ *   the user view is the `vault` value contained in VaultKit.
  *
- * A note on naming convention:
- * - `Pre` is used as a postfix for any mutable value retrieved *before* an
- *    `await`, to flag values that must used very carefully after the `await`
- * - `new` is a prefix for values that describe the result of executing a
- *   transaction; e.g., `debt` is the value before the txn, and `newDebt`
- *   will be value if the txn completes.
- * - the absence of one of these implies the opposite, so `newDebt` is the
- *   future value fo `debt`, as computed based on values after any `await`
+ *   A note on naming convention:
+ *
```

Comment:
Did it insert a blank line here? Why?

Reply:
Yes. I think because list items followed.